README.md

```diff
@@ -1 +1,167 @@
 
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+  - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is a project that shows how to set up linter configurations, and beginner html file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML5</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML5</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS3</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS3</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Displays Hello Microverse]** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ 1. Code editor , for example Visual Studio Code
+ 2. Git installed on your pc. 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with git commands such as git clone.
+
+
+
+### Usage
+
+To run the project, click on  "Go live" at the bottom left of your VS Code.
+
+
+
+### Run tests
+
+To run tests, right click on index.html to view your page on web browser.
+
+### Deployment
+
+You can deploy this project using:
+
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please give an explanation of how your project can be cloned in a local machine with your repository link, also brief step by step according to the readme microverse template
`Setup details:`
`usage details:`
`Install:`
`Run tests:`

The following :point_down: is an example of  `Setup`, `Usage`, `Install` and `Run tests` sections:

![Screenshot from 2023-06-02 14-28-20](https://github.com/jerymey/Hello-Microverse/assets/45226848/3a719127-18c4-46c2-81b1-be63c1465898)

Reply:
@aamirkhan2478 I made all the changes on the readme file and i already pushed it, please can you review it before i submit it again?